doc/md/cloud/setup-ci.mdx

```diff
@@ -0,0 +1,322 @@
+Select the repositories you want to grant access to and click "Install and Authorize".
+
+![](https://atlasgo.io/uploads/cloud/ci-guide/create-dir-4.png)
+
+### Step 3: Configure the Migration Directory
+
+Fill the rest of the form with the relevant configuration details:
+* **Directory Path** - The path to the directory containing your migrations relative to your repository root.
+* **Directory Name** - An identifier for the directory. This will be used to identify the directory in the Atlas Cloud and APIs.
+* **Migration Engine** - The format of the migrations in the directory. Atlas works best with migration directories in its
+  format, but also supports `golang-migrate`, Flyway and Liquibase.
+* **Database** - the database engine for the target database.
+
+![](https://atlasgo.io/uploads/cloud/ci-guide/create-dir-5.png)
+
+Click "Connect Directory" to finish the process.
+
+### Step 4: Test the Integration
+
+After connecting your directory, let's create a pull request with a destructive
+change to make sure that Atlas Cloud catches it.
+
+Create a new branch:
+
+```
+git checkout -b test-branch
+```
+
+Create a new migration file:
+
+```
+atlas migrate new --edit destructive
+```
+
+Add the following SQL to the migration file and save:
+
+```sql
+DROP TABLE posts;
+```
+
+Commit the changes and push them to GitHub:
+
+```
+git add .
+git commit -m "add destructive migration"
+git push origin test-branch
+```
+
+Create a pull request from the `test-branch` branch to your main branch.
+
+After a few seconds, Atlas Cloud will pick up the changes and run CI against
+them. Shortly after, you will see a comment on your pull request with the
+results of the CI run:
+
+![](https://atlasgo.io/uploads/cloud/ci-guide/app-ci.png)
+
+Amazing! Atlas Cloud caught the destructive change and prevented it from being
+accidentally merged to your main branch.
+
+To see a more detailed report of the CI run, click on the "Full Report on Atlas Cloud"
+link in the comment:
+
+![](https://atlasgo.io/uploads/cloud/ci-guide/app-cloud-report.png)
+
+
+## Push (GitHub Action) Workflow
+
+### Step 1: Create a Bot Token for Atlas Cloud
+
+In order to report the results of your CI runs to Atlas Cloud, you will need to
+create a bot token for Atlas Cloud to use. Follow [these instructions](/bot.mdx)
+to create a token and save it somewhere safe.
+
+### Step 2: Install the Atlas GitHub CLI Extension
+
+To streamline the process of configuring the GitHub Action, we've created a
+GitHub CLI extension that will do most of the work for you:
+
+1. Make sure you have the GitHub CLI installed.
+
+   ```shell
+   brew install gh
+   ```
+   See [here](https://cli.github.com/manual/installation) for more installation options.
+
+2. Install the Atlas GitHub CLI extension:
+
+   ```shell
+   gh extension install ariga/atlas
+   ```
+
+### Step 3: Configure the GitHub Action
+
+1. Make sure you have the necessary permissions to configure your action:
```

Comment:
```suggestion
1. **Permissions** - Make sure you have the necessary permissions to configure your action:
```

Reply:
I find that making the list of steps easy to remember makes absorbing the details of each list item much easier later. Hence, I made the list items 1. Permissions, 2. Root, 3. Configure so that anyone reading the tutorial can keep track of the scope of what they're doing.